plugins/generic/usageEvent/UsageEventPlugin.inc.php

```diff
@@ -286,6 +286,18 @@ function _buildUsageEvent($hookName, $args) {
 			$request, null, $canonicalUrlPage, $canonicalUrlOp, $canonicalUrlParams
 		);
 
+		// Make sure we log the server name and not aliases.
+		$configBaseUrl = Config::getVar('general', 'base_url');
+		$requestBaseUrl = $request->getBaseUrl();
+		if ($requestBaseUrl !== $configBaseUrl) {
+			// Make sure it's not an url override (no alias on that case).
+			if (!in_array($requestBaseUrl, Config::getContextBaseUrls()) && 
+					$requestBaseUrl !== Config::getVar('general', 'base_url[index]')) {
```

Comment:
ok, so in both cases the config base_url will be different compared to the HTTP_HOST. Then the check will continues, is there anyother base_url setting that matches the HTTP_HOST? No, so the final logged base url will be the config base_url...